NightHelper.ttslua

```diff
@@ -316,8 +316,6 @@ function setNightOrder(night)
                         table.insert(order, {char = 'Ambusher Info', color = v2.color})
                     elseif v2.char == 'Bogman' and night == 'first' then
                         table.insert(order, {char = 'Bogman Info', color = v2.color})
-                    elseif v2.char == 'Inquisitor' and night == 'other' then
-                        table.insert(order, {char = 'Inquisitor Info', color = v2.color})
```

Comment:
I was confused why the "Inquisitor" character I created wasn't showing up in the standard night order for a while. I couldn't see anything that references this character, so I believe it's safe to remove this special case.